raster/Makefile

```diff
@@ -44,7 +44,6 @@ SUBDIRS = \
 	r.latlong \
 	r.lake \
 	r.li \
-	r.mapcalc \
```

Comment:
Why remove `r.mapcalc`? I was having an issue building because the previous r.mapcalc dist wasn't getting removed with `make distclean`, so I needed to manually remove it.

Reply:
This is needed for the build not to error, but I didn't notice the issue with distclean. I will keep the change to pass the CI (which does not care about distclean), but I will open another PR to fix this.